tensorboard/webapp/header/plugin_selector_component.scss

```diff
@@ -53,6 +53,24 @@ mat-option {
   width: 100%;
 }
 
+// These styles exist in `mat-toolbar` already, but the combination of the
```

Comment:
Thanks. Would you mind putting a note like:
// TODO(tensorboard-team): Can probably remove after migrating away from Legacy Material components.

Reply:
Yep, done 👍